pkg/core/app/util.go

```diff
@@ -86,13 +89,25 @@ func getInode(pid int) (uint64, error) {
 	return i, nil
 }
 
-func IsDetachMode(command string) bool {
+func IsDetachMode(command string, kind utils.CmdType) error {
 	args := strings.Fields(command)
+
+	if kind == utils.DockerStart {
+		flags := []string{"-a", "--attach", "-i", "--interactive"}
+
+		for _, arg := range args {
+			if slices.Contains(flags, arg) {
```

Comment:
I think it should be `!slices.Contains(flags,arg)`, and you should log `fmt.Errorf("docker start require --attach/-a or --interactive/-i flag")` here and return true that it is in detachMode.

Reply:
Yes I made this error while refactoring changes.